sql/src/main/java/io/crate/executor/transport/AlterTableOperation.java

```diff
@@ -86,18 +88,24 @@
     private final TransportActionProvider transportActionProvider;
     private final Provider<TransportOpenIndexAction> transportOpenIndexActionProvider;
     private final Provider<TransportCloseIndexAction> transportCloseIndexActionProvider;
+    private final TransportRenameTableAction transportRenameTableAction;
+    private final TransportIndicesAliasesAction transportIndicesAliasesAction;
     private final SQLOperations sqlOperations;
 
     @Inject
     public AlterTableOperation(ClusterService clusterService,
                                TransportActionProvider transportActionProvider,
                                Provider<TransportOpenIndexAction> transportOpenIndexActionProvider,
                                Provider<TransportCloseIndexAction> transportCloseIndexActionProvider,
+                               TransportRenameTableAction transportRenameTableAction,
```

Comment:
there is no need to use providers here, just creating a PR to change that on master